lib/foreman_remote_execution/engine.rb

```diff
@@ -107,6 +107,10 @@ class Engine < ::Rails::Engine
       end
       Bookmark.send(:include, ForemanRemoteExecution::BookmarkExtensions)
       HostsHelper.send(:include, ForemanRemoteExecution::HostsHelperExtensions)
+
+      # We need to explicitly force to load the Task model due to Rails loader
+      # having issues with resolving it to Rake::Task otherwise
+      require_dependency 'foreman_tasks/task'
```

Comment:
this feels like a hack? shouldn't rails initialization phase allow you to specify it post rails bootup?